app/index-electron.jsx

```diff
@@ -22,11 +23,16 @@ if (module.hot && isDev) {
 
 
 const stores = {
-    video: new VideoStore(),
-    win: new WinStore()
+    video: new VideoStore({ events: eventsHandler }),
+    win: new WinStore({ events: eventsHandler })
 };
 
-stores.win.startLivePeer();
+
+setTimeout(() => {
```

Comment:
Is there some sort of callback or promise that could execute this function?

Reply:
probably fixed with all the cleaning I'll give a try and remove it if it works

Reply:
Right on. Let me know how it goes :)